addons/netfox/rollback/rollback-synchronizer.gd

```diff
@@ -14,17 +14,28 @@ class_name RollbackSynchronizer
 ## Turning this off is recommended to save bandwidth and reduce cheating risks.
 @export var enable_input_broadcast: bool = true
 
+## Unlike state diffs which apply project-wide, input-diffs are often undesired because they have a performance cost
+## because at each receival, there is a forloop to check against every property. And this forloop is for every player input received!
+## This is often needless for inputs, because in most games, inputs are different (movement + aim/mouse position)
+## Set to true if you have many input properties which are not updated often.
+## Set to false if your input properties are few and often update (see forest-brawl example, being movement and mouse position)
+#@export var enable_input_diffs: bool = false
```

Comment:
I was about to add input diffs as well, but I don't want this PR to get so big that it won't get reviewed (e.g. much time to test, or risky), feel free to remove.

Reply:
As mentioned in the README, please implement a single feature in a single PR. Removed this part.